tests/parsers/test_orca.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+"""Tests for the :class:`aiida_orca.parsers.OrcaBaseParser` parser."""
+
+
+def test_orca_default(
+    aiida_localhost, generate_calc_job_node, generate_parser, generate_inputs_orca, data_regression, num_regression
+):
+    """Test the ``default`` output example."""
+    name = 'default'
+    entry_point_calc_job = 'orca.orca'
+    entry_point_parser = 'orca_base_parser'
+
+    node = generate_calc_job_node(entry_point_calc_job, aiida_localhost, name, generate_inputs_orca())
+    parser = generate_parser(entry_point_parser)
+    results, calcfunction = parser.parse_from_node(node)
+
+    assert calcfunction.is_finished, calcfunction.exception
+    assert calcfunction.is_finished_ok, calcfunction.exit_message
+    assert 'relaxed_structure' in results
+    assert 'output_parameters' in results
+
+    structure_attributes = results['relaxed_structure'].attributes
+    structure_cell = [item for sublist in structure_attributes.pop('cell') for item in sublist]
```

Comment:
ORCA doesn't do periodic calculations, so there should be no cell, or a default cell. So I don't think we should check it here.

Reply:
Removed

Reply:
Note that apparently the parsed output data has a structure that actually _does_ define a cell. If this is non-sensical, maybe the parser should be updated to not create this cell in the output structure?

Reply:
@sphuber It looks like removing this line broke the test for AiiDA 1.6. I think you still need to pop the `cell` attribute (but not test it), since before AiiDA 2.0, the `StructureData` nodes required non-zero cell even for non-periodic structures. Which is why I think you might have seen it there. 

The current parser uses pymatgen to read an XYZ file from ORCA, which definitely doesn't define any cell. In the other PR I am getting rid of pymatgen anyway so we don't need to pursue this here.

Reply:
Popped the `cell` and updated the reference file